### ISSUE TITLE
Pulmonary air embolism

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6544]
**Preferred term label:** Pulmonary air embolism

**Synonyms**
Venous air embolism

**Definition (free text, please give PubMed ID)**

Venous air embolism is a consequence of air being introduced into the venous circulation, and subsequently to the right heart, and pulmonary circulation. Small amounts of air in pulmonary circulation can be removed by gas diffusion across the arteriolar wall into the alveoli, amounts of gas exceeding 50 ml can cause pulmonary outflow tract obstruction with or without concomitant arterial embolisation PMID 9172037
  

**Parent term (use hpo.jax.org/app)**
Pulmonary embolism HP:0002204

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by eswietlik** [comment]
Sorry, submitted twice.

**Comment by pnrobinson** [comment]
duplicate with https://github.com/obophenotype/human-phenotype-ontology/issues/6545
